### PR TITLE
PLAT-8490: Report inconsistent frame rate

### PR DESCRIPTION
**Pull request by gadyaari**

### CODE REVIEW

File: common/config/config.json.template [hunk 1]
```diff
@@ -69,7 +69,8 @@
       "keyFrameIntervalDiffAllowedInMSec" : 200,
       "maxFramesPerSeconds" : 30,
       "maxRecommendedKeyFrameInterval" : 4,
-      "minRecommendedKekFrameInterval" : 0.75
+      "minRecommendedKekFrameInterval" : 0.75,
```
Comment (kalturaguy) [inline common/config/config.json.template]: what is Kek?

File: lib/entry/FlavorDownloader.js [hunk 2]
```diff
@@ -392,7 +394,10 @@ var FlavorDownloader = (function() {
     function diagnosticsPushData(mp4Data) {
         this.mediaInfo.resolution = mp4Data.metaData.resolution;
         this.mediaInfo.bitrate_kbps = mp4Data.metaData.bitrate_kbps;
-        this.mediaInfo.framesPerSecond = mp4Data.metaData.framerate;
+        if (this.mediaInfo.framerateArray.length === numOfFrameRateSamples) {
```
Comment (kalturaguy) [inline lib/entry/FlavorDownloader.js]: >=

File: lib/Diagnostics/DiagnosticsAnalyzer.js [hunk 3]
```diff
@@ -22,6 +22,8 @@ class DiagnosticsAnalyzer extends EventEmitter{
         this._entryId = entryId;
         this._logger = loggerModule.getLogger('DiagnosticsAnalyzer', '[' + entryId + '] ');
         this._bitrateDiffPercentageAllowed = config.get('diagnostics').bitrateDiffPercentageAllowed;
+        this._frameRateMedianMap = new Map();
```
Comment (kalturaguy) [inline lib/Diagnostics/DiagnosticsAnalyzer.js]: change Median to avg

File: common/config/config.json.template (hunk 1 above)
Comment (gadyaari) [reply common/config/config.json.template]: done

File: lib/Diagnostics/DiagnosticsAnalyzer.js (hunk 3 above)
Comment (gadyaari) [reply lib/Diagnostics/DiagnosticsAnalyzer.js]: done

File: lib/entry/FlavorDownloader.js (hunk 2 above)
Comment (gadyaari) [reply lib/entry/FlavorDownloader.js]: ?...

**Pull request merged**